src/promptflow-evals/promptflow/evals/evaluate/_batch_run_client/proxy_client.py

```diff
@@ -5,10 +5,11 @@
 
 import numpy as np
 
+from promptflow._utils.utils import get_int_env_var
```

Comment:
This is from a private namespace and in my experience if it changes and breaks things. If it not a big functionality can be add it to evals utils

Reply:
Cool, I'll copy the method over to evals utils. It's a general method.